src/components/TicketStatusFilter.tsx

```diff
@@ -2,26 +2,32 @@ import { Checkbox, FormControl, InputLabel, MenuItem, Select, SelectChangeEvent,
 import { useTranslation } from 'react-i18next';
 import { useHistory } from 'react-router-dom';
 import { TicketSearchParam } from '../api/searchApi';
-import { TicketStatus, ticketStatusValues } from '../types/publication_types/ticket.types';
+import { TicketStatus } from '../types/publication_types/ticket.types';
 import { dataTestId } from '../utils/dataTestIds';
 
 const labelId = 'status-filter-select';
 
-export const TicketStatusFilter = () => {
+interface TicketStatusFilterProps {
+  options: TicketStatus[];
+}
+
+export const TicketStatusFilter = ({ options }: TicketStatusFilterProps) => {
   const { t } = useTranslation();
   const history = useHistory();
   const searchParams = new URLSearchParams(history.location.search);
-  const selectedStatuses = (searchParams.get(TicketSearchParam.Status)?.split(',') ?? []) as TicketStatus[];
+  const currentStatusFilter = (searchParams.get(TicketSearchParam.Status)?.split(',') ?? []) as TicketStatus[];
+  const selectedOptions = currentStatusFilter.filter((status) => options.includes(status));
+  const otherSelectedStatuses = currentStatusFilter.filter((status) => !options.includes(status));
```

Comment:
Kunne effektivisert dette til å loope gjennom arrayet bare en gang med en `.reduce()` eller noe sånt, men føler det øker kompleksiteten for mye. Siden det er maks 4 elementer her føler jeg lesbarhet er mye viktigere enn effektivisering